docs/docs/channels/sms/index.md

```diff
@@ -7,3 +7,4 @@ To read a provider specific documentation:
 - [AWS SNS](/channels/sms/sns)
 - [SMS77](/channels/sms/SMS77)
 - [TWILIO](/channels/sms/twilio)
+- [TELNYX](https://github.com/novuhq/novu/blob/84682038213c26c526f577c3668e028f6da4936a/docs/docs/channels/sms/telnyx.md)
```

Comment:
```suggestion
- [TELNYX](/channels/sms/telnyx.md)
```

Reply:
- I tried that, but in GitHub md the link is giving 404 
![image](https://user-images.githubusercontent.com/58129377/195548544-b9d4f688-e53c-45c3-961d-02a5e1a2453d.png)
- Works well though for website. 

So I was confused on that matter.